packages/block-editor/src/hooks/custom-fields.js

```diff
@@ -105,7 +105,7 @@ const withInspectorControl = createHigherOrderComponent( ( BlockEdit ) => {
 														},
 													},
 												},
-												[ attributeName ]: undefined,
```

Comment:
Setting this to undefined will remove the block content in the editor in the case of the Image block because an `<img>` without `src` is invalid markup. However, setting the `url` to `''` (empty string) works.